diff --git a/src/UmbracoProject/Views/home.cshtml b/src/UmbracoProject/Views/home.cshtml
new file mode 100644
index 0000000..a173389
--- /dev/null
+++ b/src/UmbracoProject/Views/home.cshtml
@@ -0,0 +1,5 @@
+﻿@using Umbraco.Cms.Web.Common.PublishedModels;
+@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage
+@{
+	Layout = null;
+}
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/document-type__84ca1fcce5ca42d0ba1a285ba2063e36.uda b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__84ca1fcce5ca42d0ba1a285ba2063e36.uda
new file mode 100644
index 0000000..42bc39c
--- /dev/null
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/document-type__84ca1fcce5ca42d0ba1a285ba2063e36.uda
@@ -0,0 +1,45 @@
+{
+  "Name": "Home",
+  "Alias": "home",
+  "DefaultTemplate": "umb://template/944428c577ac4acbbcfa46783ee85f36",
+  "AllowedTemplates": [
+    "umb://template/944428c577ac4acbbcfa46783ee85f36"
+  ],
+  "HistoryCleanup": {},
+  "Icon": "icon-notepad",
+  "Thumbnail": "folder.png",
+  "Permissions": {
+    "AllowedAtRoot": true,
+    "AllowedChildContentTypes": []
+  },
+  "CompositionContentTypes": [],
+  "PropertyGroups": [
+    {
+      "Key": "5cf07481-1185-4806-b230-a03282a77e20",
+      "Name": "Content",
+      "Alias": "content",
+      "PropertyTypes": [
+        {
+          "Key": "28f52a44-4503-4135-9f37-df858c26293c",
+          "Alias": "body",
+          "DataType": "umb://data-type/ca90c9500aff4e72b976a30b1ac57dad",
+          "Name": "Body"
+        }
+      ]
+    }
+  ],
+  "PropertyTypes": [],
+  "Udi": "umb://document-type/84ca1fcce5ca42d0ba1a285ba2063e36",
+  "Dependencies": [
+    {
+      "Udi": "umb://data-type/ca90c9500aff4e72b976a30b1ac57dad",
+      "Ordering": true
+    },
+    {
+      "Udi": "umb://template/944428c577ac4acbbcfa46783ee85f36",
+      "Ordering": true
+    }
+  ],
+  "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContentType.DocumentTypeArtifact",
+  "__version": "14.1.1"
+}
\ No newline at end of file
diff --git a/src/UmbracoProject/umbraco/Deploy/Revision/template__944428c577ac4acbbcfa46783ee85f36.uda b/src/UmbracoProject/umbraco/Deploy/Revision/template__944428c577ac4acbbcfa46783ee85f36.uda
new file mode 100644
index 0000000..136a39b
--- /dev/null
+++ b/src/UmbracoProject/umbraco/Deploy/Revision/template__944428c577ac4acbbcfa46783ee85f36.uda
@@ -0,0 +1,14 @@
+{
+  "Name": "Home",
+  "Alias": "home",
+  "File": "umb://template-file/home.cshtml",
+  "Udi": "umb://template/944428c577ac4acbbcfa46783ee85f36",
+  "Dependencies": [
+    {
+      "Udi": "umb://template-file/home.cshtml",
+      "Ordering": true
+    }
+  ],
+  "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.TemplateArtifact",
+  "__version": "14.1.1"
+}
\ No newline at end of file

